learning_loop_node/detector/detector_node.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, name: str, detector: DetectorLogic, uuid: Optional[str] = Non
         self.log.info('Using %s/%s', self.organization, self.project)
         self.operation_mode: OperationMode = OperationMode.Startup
         self.connected_clients: List[str] = []
+        self.previous_state: Optional[str] = None
```

Comment:
```suggestion
```

Reply:
True.. didn't clean up  after moving these to parent